tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderStatusTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections.Generic;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Ordering.Domain.UnitTests
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    internal sealed class OrderStatusTests
+    {
+        [Test]
+        public void List_ReturnsExpectedList()
+        {
+            var actual = OrderStatus.List();
+
+            var expected = new List<OrderStatus>
+            {
+                OrderStatus.Incomplete,
+                OrderStatus.Complete,
+            };
+
+            actual.Should().BeEquivalentTo(expected);
+        }
+
+        [Test]
+        public void FromId_OrderStatusId_ReturnsExpectedType()
+        {
+            var actual = OrderStatus.FromId(1);
+
+            actual.Should().Be(OrderStatus.Complete);
+        }
+
+        [Test]
+        public void FromId_UnknownOrderStatusId_ReturnsNull()
+        {
+            var actual = OrderStatus.FromId(10);
+            actual.Should().BeNull();
+        }
+
+        [TestCase("Complete")]
+        [TestCase("complete")]
+        [TestCase("comPLete")]
```

Comment:
Do you not want tests for the order being incomplete?

Reply:
This test was really just to cover the FromName method & I used the complete as an example. Didn't feel the need to add the incomplete one. Though you have highlighted that the name of the method is wrong. Would you like me to add an incomplete test as well, don't mind either way?

Reply:
Ahhh, sorry thats my misunderstanding of the test. Dw

Reply:
No apology needed, it was the right question to ask 👍